service/openapi.yaml

```diff
@@ -279,3 +314,33 @@ components:
     errorDetails:
       type: string
       description: An error description for logging or other needs
+    metadataResponse:
+      description: OFREP metadata response
+      properties:
+        name:
+          type: string
+          description: name of the flag management system
+          examples:
+            - flagd
+            - go-feature-flag
+        features:
```

Comment:
`minPollingInterval` from `configurations` below is clearly connected with one of these "features"? Is it feasibile to come up with a format to pass the `configuration` as part of the `feature`?

Reply:
Would something like this example look better to you?

```json
{
   "name": "flagD",
   "features": {
      "cacheInvalidation": {
         "supported": ["POLLING"],
         "polling": {
            "minPollingInterval": 1000
         }
      }
   }
}
```

Reply:
I like the first one, supporting the 2 syntax with the same key will be not super easy in some languages that are not supporting combine types.

I can update the openAPI specs with the 1st proposition if you're ok.

Reply:
yeah, I like the first one as well 👍

Reply:
Not a fan of using `features` here, as most providers call themselves `Feature Management Platforms` now, and many systems have the concept of `features` as a top-level objects that will cause confusion here. I'd stick to calling this `configuration` / `options` something in that line.